util/test/testUtil.h

```diff
@@ -50,4 +50,14 @@ void valuecheck(double a, double b, double tolerance = 1e-8)
   }
 }
 
+template <typename T>
+void valuecheck(T const & a, T const & b)
```

Comment:
this is just exacerbating the problem.  instead of adding this, can you just make the change to using the drakeUtil.h valuecheck (moving it to testUtil.h is aok with me)?